com.github.you_win.VPupPr.yml

```diff
@@ -0,0 +1,50 @@
+id: com.github.you_win.VPupPr
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: OpenSeeFaceGD.x86_64
+finish-args:
+    - "--socket=x11"
+    - "--share=ipc"
+    - "--share=network" # to set up the deps for openseeface
+    - "--device=all"
+    - "--socket=pulseaudio"
+    - "--filesystem=home:ro"
+modules:
+
+  # --- Puppeteer ---
+  - name: puppeteer
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin/{resources,OpenSeeFaceFolder}
+      - install -Dm 644 -t /app/share/applications com.github.you_win.VPupPr.desktop
+      - |
+        for icon in 16 24 32 48 64 128 256; do
```

Comment:
for the record, only 128x128 is required. So I would reduce the number of icons here, and if you want that many, make sure upstream provide them instead.

Reply:
I've submitted the icons to upstream already, so I'd say this suggestion is already covered. I'll consider lowering it to 128x128, but I don't see any reason to.

Reply:
I've removed them and just used `128`, if I get any complaints about it looking blurry, I will add back `256`.